client/pages/Assessment/SectionWrapper/SectionWrapper.tsx

```diff
@@ -38,7 +38,7 @@ const SectionWrapper: React.FC<Props> = (props) => {
       AssessmentSectionActions.getTableSections({
         assessmentName,
         cycleName,
-        section,
+        sections: [section],
```

Comment:
One thought was to omit 'sections' param and return all sections if sections not specified
This could be done on upper level to optimize performance

Reply:
better the way it is ;)